### DISCUSSION TITLE
[Suggestion] Increase collaboration with other projects

### DISCUSSION THREAD

**Discussion by prk0ghy** [TagStudioDev/TagStudio#243] [Ideas]
Hey @CyanVoxel , Hey @Loran425 ,

first of all i would like to thank you for this amazing project.
Here is a list of projects that work on similar ideas. I would suggest combining some of the efforts.
https://github.com/stars/prk0ghy/lists/duplicate-utils-merge
Sorry for the overly short text, i will add more asap.

**Comment by Loran425** [comment]
Thanks for the kind message, TagStudio is definitely @CyanVoxel's project I'm just happy to get the chance to assist where I can and learn along the way.

**Comment by CyanVoxel** [comment]
There's already some (poorly documented) support for importing duplicate file lists generated from dupeGuru, however #23 mentions that dupeGuru hasn't had a new release in a couple years and suggests [Czkawka](https://github.com/qarmin/czkawka) as a potential alternative, however I haven't personally tried it out.
I'll take a look at the other items in your list as well!

**Reply by PriyaanG** [reply]
### Combine Chat Rocketspace and TagStudio Efforts

Hey @CyanVoxel,
I am new to web development. I am working on a chat app called Chat Rocketspace which also wishes to be a open source and privacy protective alternative to google chat, meet, etc. If we could combine the efforts, I think it would provide a great alternative to google products for the open source and privacy concerned community. I would also wish to start mobile and web development for tagstudio using HTML, CSS, and Javascript with React and Expo. I would just want your input on if I should start now or after the full release.

**Reply by prk0ghy** [reply]
@CyanVoxel  should i go and try to contact the devs of the other projects?

**Reply by CyanVoxel** [reply]
> ### Combine Chat Rocketspace and TagStudio Efforts
> Hey @CyanVoxel, I am new to web development. I am working on a chat app called Chat Rocketspace which also wishes to be a open source and privacy protective alternative to google chat, meet, etc. If we could combine the efforts, I think it would provide a great alternative to google products for the open source and privacy concerned community. I would also wish to start mobile and web development for tagstudio using HTML, CSS, and Javascript with React and Expo. I would just want your input on if I should start now or after the full release.

There's currently a Rust/Tauri [fork](https://github.com/TagStudioDev/TagStudioRusted) that's looking to accomplish a similar goal, however I'm not directly involved in the development of that one. Being an open source project though it's up to you if you want to try something out yourself. The only planned major structural change to the program is the [database migration](https://github.com/TagStudioDev/TagStudio/blob/main/doc/updates/db_migration.md) which the Tauri fork is working on in tandem.

**Reply by CyanVoxel** [reply]
> @CyanVoxel should i go and try to contact the devs of the other projects?

There's no need at the moment, since built-in file duplication detection that overlaps with existing programs is currently a low priority compared to other areas of development. And for now there can be continued support for importing results files from these existing programs.
